CITATION.cff

```diff
@@ -19,17 +19,17 @@ authors:
     family-names: Pincelli
     orcid: 'https://orcid.org/0000-0003-2692-2540'
   - given-names: Arora
-    family-names: Abeer
+    family-names: Abeer 
```

Comment:
family name and given name needs to be flipped and trailing whitespace removed.

https://pc.fhi-berlin.mpg.de/people/